fastestimator/pipeline.py

```diff
@@ -482,6 +488,17 @@ def __enter__(self) -> Union[DataLoader, tf.data.Dataset]:
             collate_fn = self.collate_fn
             if collate_fn is None and self.pad_value is not None:
                 collate_fn = self._pad_batch_collate
+            # Default ExapandDataset Sampler
+            if self.ctx_steps_per_epoch is not None and self.ctx_mode in ["train", "eval"]:
```

Comment:
no need to check ctx_mode here. maybe they only want to use 3 steps of test data...